src/components/films-catalog/films-catalog.tsx

```diff
@@ -0,0 +1,36 @@
+import FilmsList from '../films-list/films-list';
+import GenresList from '../genres-list/genres-list';
+import { useAppDispatch, useAppSelector } from '@/hooks';
+import { chooseGenreAction, resetGenreAction} from '@/store/action';
+import { Genres } from '@/utils/const';
+import { useState } from 'react';
+
+
+export default function FilmsCatalogue() {
```

Comment:
нейминг такой же как и у файла

```suggestion
export default function FilmsCatalog() {
```